### PR TITLE
Adds support for evaluating new sub layers within an existing intent layer

### PR DESCRIPTION
**Pull request by janaab11**

### CODE REVIEW

**Comment by janaab11** [comment]
@greed2411 can you test this branch to see if the new cli runs. if you run into errors, ping me - the code is very messy and will need to be refactored if there are errors. else, you can merge since @ltbringer needs this for the metabase tool

File: eevee/metrics/classification.py [hunk 1]
```diff
@@ -96,3 +96,108 @@ def intent_report(
...
+
+    # aliasing preds
+    col = "intent_y"
+    intents_dict = {value: key for key, values in intent_layers.get(col).items() for value in values}
+    df[col] = df[col].apply(lambda intent: intents_dict.get(intent, intent))
+
+    #aliasing trues
+    col = "intent_x"
+    intents_dict = {value: key for key, values in intent_layers.get(col).items() for value in values}
+    df["{}-alias".format(col)] = df[col].apply(lambda tag: intents_dict.get(col).get(tag, tag))
```
Comment (greed2411) [inline eevee/metrics/classification.py:120]: it is crashing here:
```
    df["{}-alias".format(col)] = df[col].apply(lambda tag: intents_dict.get(col).get(tag, tag))
AttributeError: 'NoneType' object has no attribute 'get'
```
where `col` is `intent_x` and `intents_dict` looks like this:

```
{'_': 'acoustic_oos',
 '_oos_': 'lexical_oos',
 'audio_channel_noise': 'acoustic_oos',
 'audio_channel_noise_hold': 'acoustic_oos',
 'audio_silent': 'acoustic_oos',
 'audio_speech_unclear': 'acoustic_oos',
 'audio_speech_volume': 'acoustic_oos',
 'background_noise': 'acoustic_oos',
 'background_speech': 'acoustic_oos',
 'ood': 'lexical_oos',
 'other_language': 'acoustic_oos',
 'partial': 'lexical_oos'}
 ```

Comment (janaab11) [reply eevee/metrics/classification.py]: fixed

**Comment by greed2411** [comment]
merging it for now.

**Pull request merged**